packages/snowflake.rb

```diff
@@ -3,53 +3,37 @@
 class Snowflake < Package
   description 'Snowflake (aka Muon) is a graphical SSH client.'
   homepage 'https://github.com/subhra74/snowflake'
-  version '1.0.4'
+  version '1.0.4-1'
```

Comment:
We shouldn't need the version bump here, and if we do, something is broken which we should fix.

Reply:
This does fix an issue with the jdk version.  It also prepares for the next update.

Reply:
I'm not saying we shouldn't upgrade the package, just that we don't need to bump the version